### PR TITLE
Activate JNLP3 protocol

### PR DESCRIPTION
**Pull request by kohsuke**
JNLP3 protocol support is live now, with some more massaging to move some of the server-side code into the remoting for better pairing.

### CODE REVIEW

**Comment by kohsuke** [comment]
cc @akshayabd @oleg-nenashev @jglick

File: src/main/java/org/jenkinsci/remoting/engine/JnlpProtocol3.java [hunk 1]
```diff
@@ -37,7 +37,7 @@
 import java.util.Properties;
 
 import static org.jenkinsci.remoting.engine.EngineUtil.*;
-import static org.jenkinsci.remoting.engine.Jnlp3Util.createChallengeResponse;
+import static org.jenkinsci.remoting.engine.Jnlp3Util.*;
```
Comment (amuniz) [inline src/main/java/org/jenkinsci/remoting/engine/JnlpProtocol3.java]: We should avoid * imports, unless on test code.

**Comment by kohsuke** [comment]
Turns out I still got some issues with this code. Please standby

**Comment by kohsuke** [comment]
With the updated push this change should br good to go.

**Pull request merged**